docs/dsl_inspec.md

```diff
@@ -285,18 +285,18 @@ control 'check-perl' do
   #require 'pry'; binding.pry;
   describe perl_out do
     its('exit_status') { should eq 0 }
-    its('stdout') { should match (/USE_64_BIT_ALL/) }
-    its('stdout') { should match (/useposix=true/) }
-    its('stdout') { should match (/-fstack-protector/) }
+    its('stdout') { should match /USE_64_BIT_ALL/ }
+    its('stdout') { should match /useposix=true/ }
+    its('stdout') { should match /-fstack-protector/ }
   end
 
   # extract an array of include directories
   perl_inc = perl_out.stdout.partition('@INC:').last.strip.split("\n")
   # ensure include directories are only writable by 'owner'
   perl_inc.each do |path|
     describe directory(path.strip) do
-      it { should_not be_writable.by('group') }
-      it { should_not be_writable.by('other') }
+      it { should_not be_writable.by 'group' }
```

Comment:
Now see...I actually like `by('group')`...style is hard...

Reply:
Well, my thought there is that if we have `.by('other')` and in other examples use the style `it { should match /joe/ }` and further `{ should be <= 3 }` vs `it { should_not be_writable.by 'group' }` then it would be hard to ping someone on `it { should match.(/joe/) }`. They would say why is the general style `x` in one place and `y` in another - I think anyway.

Reply:
But I think what @jerryaldrichiii gut is reacting to is the `.` .... i.e. if we have made the interface `it { should_not be_writable_by 'group' }` then his normal habit of thinking of `.by('other')` as a function is is calling off of `be_writable` would serve him well. And really I can see his point in this gut instinct.

Reply:
Yeah fully agree @aaronlippold , that was my problem too. I was pondering making an exception for chaining, but then it's tricky with the other argument. we do want to avoid parentheses on the simple cases + then be consistent, so it hit chaining in the process. But we could make an exception here...

Reply:
I think an exception should be made for chaining. I worry that people who are not developers will be confused by:

```
it { should be_writable.by 'group' }
```

because the `.` looks like a typo for a `_` to the uninitiated.

I much prefer making an exception for chaining and then adding aliases when possible (e.g. `be_writable_by`)